services/uptimeobserver/uptimeobserver-base.js

```diff
@@ -0,0 +1,59 @@
+import Joi from 'joi'
+import { BaseJsonService, InvalidParameter, InvalidResponse } from '../index.js'
+
+const errorResponse = Joi.object({
+  status: Joi.string().required(),
+  error: Joi.string(),
+}).required()
+
+const monitorResponse = Joi.object({
+  status: Joi.string().required(),
+  friendlyName: Joi.string().required(),
+  lastExecution: Joi.string().isoDate().required(),
```

Comment:
Correct me if I'm wrong, but it seems `friendlyName` & `lastExecution` are never used.
Let's remove them to save up on resources and avoid invalid response for info we do not need.